resources/js/Components/Tables/AdminBookingRequestTable.vue

```diff
@@ -0,0 +1,246 @@
+              </div>
+            </div>
+          </div>
+
+          <div class="flex flex-row">
+            <div class="flex flex-col flex-1 py-2 px-3">
+              <div><h2>Date Created</h2></div>
+              <div class="flex flex-col">
+                <jet-label for="start_time" value="Before" />
+                <jet-input
+                  id="start_time"
+                  type="datetime-local"
+                  class="mt-1 block w-full"
+                  v-model="jsonFilters.date_range_start"
+                  autofocus
+                />
+              </div>
+              <div class="flex flex-col">
+                <jet-label for="end_time" value="After" />
+                <jet-input
+                  id="end_time"
+                  type="datetime-local"
+                  class="mt-1 block w-full"
+                  v-model="jsonFilters.date_range_end"
+                  autofocus
+                />
+              </div>
+            </div>
+          </div>
+        </div>
+
+      </template>
+
+      <template #footer>
+        <jet-button
+          class="ml-2"
+          @click.native="advancedFilters()"
+        >
+          Filter
+        </jet-button>
+        <jet-secondary-button @click.native="toggleAdvancedFilters">
+          Close
+        </jet-secondary-button>
+      </template>
+    </jet-dialog-modal>
+  </div>
+</template>
+<script>
+
+import JetDialogModal from "@src/Jetstream/DialogModal";
+import JetSecondaryButton from "@src/Jetstream/SecondaryButton";
+import JetDropdown from "@src/Jetstream/Dropdown";
+import JetDropdownLink from "@src/Jetstream/DropdownLink";
+import JetButton from "@src/Jetstream/Button";
+import JetLabel from "@src/Jetstream/Label";
+import JetResponsiveNavLink from "@src/Jetstream/ResponsiveNavLink"
+import JetInput from "@src/Jetstream/Input";
+import MultiSelect from "@src/Components/Form/MultiSelect";
+
+export default {
+  name: "AdminBookingRequestTable",
+  props: {
+    bookings: {
+      type: Array,
+      default: function () {
+        return []
+      },
+    },
+    statuses:{
+      type: Array,
+      default: function () {
+        return []
+      },
+    },
+    reviewers: {
+      type: Array,
+      required: true,
+    },
+  },
+  components: {
+    JetResponsiveNavLink,
+    JetDropdownLink,
+    JetDropdown,
+    JetButton,
+    JetDialogModal,
+    JetSecondaryButton,
+    JetLabel,
+    JetInput,
+    MultiSelect
+  },
+  data() {
+    return {
+      filter: '',
+      showFilterModal: false,
+      selected: {},
+      options: {},
+      jsonFilters: {
+        status_list: [],
+        date_range_start: null,
+        date_range_end: null,
+        data_reviewers: []
+      }
+    }
+  },
+  mounted(){
+    const status_zip = {};
+    for (var i = 0; i < this.statuses.length; i++) {
+      status_zip[this.statuses[i]] = false;
+    }
```

Comment:
haven't seen this type of setup before, I like it. I was using reduce before..

Reply:
Yes similar to the python zip function implementation, tried to replicate in JS